neural_compressor/experimental/benchmark.py

```diff
@@ -249,7 +251,7 @@ def config_instance(self):
                 multi_instance_cmd += '{} \n'.format(instance_cmd)
 
         multi_instance_cmd += 'wait' if sys.platform in ['linux'] else ''
-        logger.info("Running command is\n{}".format(multi_instance_cmd))
+        print("Running command is\n{}".format(multi_instance_cmd))
```

Comment:
why use print but not logger?

Reply:
for debug only, will change update soon.